src/Symfony/Component/HttpFoundation/Tests/FileBagTest.php

```diff
@@ -60,6 +64,26 @@ public function testShouldSetEmptyUploadedFilesToNull()
         $this->assertNull($bag->get('file'));
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Passing a size as 4th argument to the constructor of "Symfony\Component\HttpFoundation\File\UploadedFile" is deprecated since Symfony 4.1 and will be unsupported in 5.0.
+     */
+    public function testShouldNotTriggerDeprecationWhenPassingSize()
```

Comment:
I think we should remove this test entirely. Not even does the test name not match the asserted output, it is also just the repetition of what we already have in the `UploadedFileTest` class.

Reply:
see 7bb538bdca0b8c80dc781e6e2d4908179bd893d1